morebits.js

```diff
@@ -2236,9 +2236,19 @@ Morebits.wiki.page = function(pageName, currentAction) {
 
 		switch (ctx.editMode) {
 			case 'append':
+				if (ctx.appendText === 'null') {
```

Comment:
```suggestion
				if (ctx.appendText === null) {
```
comparing with string null?

Reply:
Brainfart 🧠 💨